src/Game.js

```diff
@@ -0,0 +1,41 @@
+export default class Game {
+    constructor() {
+        // this.alive = true;
+        this.player1;
+        this.player2;
+        this.wordToGuess = '';
+        this.attemptedGuesses = []; // All of these will be displayed as guesses
+        this.maxWrongAttempts = 6;
+        this.wrongGuesses = 0;
+        this.correctGuesses = []; // These will be displayed as correct guesses only
+    }
+    reviewAttempt(guess) {
+        this.attemptedGuesses.push(guess);
+        if (!this.wordToGuess == guess || !this.wordToGuess.includes(guess)) {
```

Comment:
I think `!this.wordGuess == guess` would be better written `this.wordToGuess !== guess`,

`!this.wordGuess` evaluate's to `true` or `false` probably will never == `guess`.

Reply:
I almost want to say it'll work like this but if it doesn't I can definitely change it

Reply:
If a variable (say `foo`) is assigned to any non-empty string (aka truthy), `!foo` will evaluate to `false`.

Unless `guess` is set to `false`,  `false == guess` will always evaluate to false.

If `foo` is an empty string, `!foo` will evaluate to `true`.

Unless `guess` is set to `true`, `true == guess` will always evaluate to false.

So, basically, this conditional will practically always evaluate to `false`, which means one of two things:

1. We shouldn't include it in the conditional since it doesn't matter 

or 2. (the most likely scenario) the conditional is written in a way that doesn't represent the the underlying logic we want it to.